packages/redux/src/exchanges/actions/resetExchangeFiltersState.ts

```diff
@@ -0,0 +1,53 @@
+import * as actionTypes from '../actionTypes.js';
```

Comment:
Since we are adding a breaking change, rename this file to `resetExchangeFilters` and change the export in `packages/redux/src/exchanges/actions/index.ts` to `resetExchangeFilters` as well.

Reply:
Done 👍